### ISSUE TITLE
Download weather data and process it.

### ISSUE THREAD

**Issue by bhavintandel** [CloudwickShared/Hadoop#50]
Write Shell script to download the weather data using ftp download. When downloading, download data for each year separately. untar and combine them into a single file (make sure you only have one header line in the data). Try doing all of it in a single shell script.

**Comment by StayerX** [comment]
How is this issue going?

**Comment by bhavintandel** [comment]
Its ready but i havent made it generic 
Should i upload current script ?

**Comment by StayerX** [comment]
Yes, GitHub has work in progress. So you should at least have the branch here, even if it is not for a pull.
